packages/framework/tests/Unit/Model/Customer/UserFactoryTest.php

```diff
@@ -172,20 +60,18 @@ private function getEncoderFactory(): EncoderFactory
     }
 
     /**
-     * @param \Shopsys\FrameworkBundle\Model\Customer\BillingAddressData|null $billingAddressData
      * @return \Shopsys\FrameworkBundle\Model\Customer\BillingAddress
      */
-    private function createBillingAddress(BillingAddressData $billingAddressData = null)
+    private function createBillingAddress()
```

Comment:
it would be more readable if these changes were in a separate commit - they are not related to the rest of the changes, right?